app/code/core/Mage/Adminhtml/Block/Cache/Grid.php

```diff
@@ -96,6 +96,15 @@ protected function _prepareColumns()
             'frame_callback' => [$this, 'decorateStatus']
         ]);
 
+        $this->addColumn('refreshed_at', array(
+            'header'    => $this->__('Refreshed At'),
```

Comment:
```suggestion
            'header'    => $this->__('Updated At'),
```

I wouldn't add a new translation string for this

Reply:
I proposed the same name of the column as "Updated At". It already exists in translations, it is used in the grid in "Index Management" page. 

I come with an additional argument, on this page we have two actions Flush and Refresh, instead of saying "Flushed At" or "Refreshed At", we only use "Updated At" which covers both.